website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Query patterns and endpoints in the dbt Cloud Discovery API"
+description: "August 2023: Learn about the upcoming deprecation of certain endpoints and query patterns in the Discovery API."
+sidebar_position: 6
+sidebar_label: "Deprecation: Certain Dicovery API endpoints and query patterns"
+tags: [Aug-2023, API]
+date: 2023-08-31
+---
+
+dbt Labs has deprecated and will be deprecating certain query patterns and replacing them with new conventions to enhance the performance of the dbt Cloud [Discovery API](/docs/dbt-cloud-apis/discovery-api). 
+
+All these changes will be in effect on _September 7, 2023_. 
+
+We understand that these changes might require adjustments to your existing integration with the Discovery API. Please [contact us](mailto:support@getdbt.com) with any questions. We're here to help you during this transition period.
```

Comment:
@cafzal : is linking to support@getdbt.com the correct one to use?

Reply:
yes